content/influxdb3/enterprise/reference/config-options.md

```diff
@@ -972,47 +972,31 @@ Defines the interval at which each replica specified in the
 
 ### Compaction
 
-- [compactor-id](#compactor-id)
-- [compact-from-node-ids](#compact-from-node-ids)
-- [run-compactions](#run-compactions)
 - [compaction-row-limit](#compaction-row-limit)
 - [compaction-max-num-files-per-plan](#compaction-max-num-files-per-plan)
 - [compaction-gen2-duration](#compaction-gen2-duration)
 - [compaction-multipliers](#compaction-multipliers)
+- [compaction-cleanup-wait](#compaction-cleanup-wait)
 - [gen1-duration](#gen1-duration)
 
 #### compactor-id
 
-Specifies the prefix in the object store where all compacted data is written.
-Provide this option only if this server should handle compaction for its own
-write buffer and any replicas it manages.
-
-| influxdb3 serve option | Environment variable                |
-| :--------------------- | :---------------------------------- |
-| `--compactor-id`       | `INFLUXDB3_ENTERPRISE_COMPACTOR_ID` |
+> [!Important]
+> This option has been superseded by using `--mode compact`. See the [mode](#mode) section for details.
 
 ---
 
 #### compact-from-node-ids
 
-Defines a comma-separated list of writer identifier prefixes from which data is
-compacted.
-
-| influxdb3 serve option      | Environment variable                           |
-| :-------------------------- | :--------------------------------------------- |
-| `--compact-from-node-ids` | `INFLUXDB3_ENTERPRISE_COMPACT_FROM_WRITER_IDS` |
+> [!Important]
+> This option has been superseded by using `--mode compact`. See the [mode](#mode) section for details.
 
 ---
 
 #### run-compactions
 
-Indicates that the server should run compactions. Only a single server should
-run compactions for a given `compactor-id`. This option is only applicable if a
-`compactor-id` is set.
-
-| influxdb3 serve option | Environment variable                   |
-| :--------------------- | :------------------------------------- |
-| `--run-compactions`    | `INFLUXDB3_ENTERPRISE_RUN_COMPACTIONS` |
+> [!Important]
+> This option has been superseded by using `--mode compact`. See the [mode](#mode) section for details.
 
 ---
```

Comment:
Same. Just remove these. If we haven't already, we should make a note in the release notes that these have been removed and replaced by the `mode` options.